### PR TITLE
[s] Cleanup escalation unsafe procs

### PR DESCRIPTION
**Pull request by Cyberboss**
Do 👏🏾 not 👏🏾 let 👏🏾 admins 👏🏾 create 👏🏾 arbitrary 👏🏾 types 👏🏾 outside 👏🏾 verbs

### CODE REVIEW

**Review by Watermelon914 — APPROVED** [review]

**Comment by Colovorat** [comment]
Fikou accident 2021

File: code/game/alternate_appearance.dm [hunk 1]
```diff
@@ -13,6 +13,9 @@ GLOBAL_LIST_EMPTY(active_alternate_appearances)
 		return
 	if(alternate_appearances && alternate_appearances[key])
 		return
+	if(!(type in subtypesof(/datum/atom_hud/alternate_appearance)))
```
Comment (TheFakeElon) [inline code/game/alternate_appearance.dm:16]: might be worth static caching the subtype list

**Comment by LemonInTheDark** [comment]
Better reason, that code was on drugs and deserved the axe

File: code/game/alternate_appearance.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/game/alternate_appearance.dm:16]: I mean 
image

**Pull request merged**